django/db/backends/base/schema.py

```diff
@@ -798,7 +799,9 @@ def _create_index_name(self, model, column_names, suffix=""):
             )
         # Else generate the name for the index using a different algorithm
         table_name = model._meta.db_table.replace('"', '').replace('.', '_')
-        index_unique_name = '_%x' % abs(hash((table_name, ','.join(column_names))))
```

Comment:
Nothing much, however there can be when python 2 is run with the -R parameter added and migrations done while it's run that way. Py3 has -R turned on per default.